src/frontends/paddle/src/op/elementwise_ops.cpp

```diff
@@ -46,6 +46,32 @@ NamedOutputs elementwise_greater_equal(const NodeContext& node_context) {
     return elementwise_ops<default_opset::GreaterEqual>(node_context);
 }
 
+NamedOutputs elementwise_floordiv(const NodeContext& node_context) {
+    auto x = node_context.get_input("X");
+    auto y = node_context.get_input("Y");
+
+    auto axis = node_context.get_attribute<int>("axis");
```

Comment:
better use the default value for 'get_attribute()' function in case it throw error for models which doesn't set this attributes

Reply:
and also add "const" if the variable's value won't change.

Reply:
done